pcsx2-qt/GameList/GameListModel.cpp

```diff
@@ -467,10 +456,13 @@ void GameListModel::loadCommonImages()
 	m_type_disc_with_settings_pixmap = QIcon(QStringLiteral(":/icons/media-optical-gear-24.png")).pixmap(QSize(24, 24));
 	m_type_exe_pixmap = QIcon(QStringLiteral(":/icons/applications-system-24.png")).pixmap(QSize(24, 24));
 	m_type_playlist_pixmap = QIcon(QStringLiteral(":/icons/address-book-new-22.png")).pixmap(QSize(22, 22));
-	m_region_eu_pixmap = QIcon(QStringLiteral(":/icons/flag-eu.png")).pixmap(QSize(42, 30));
-	m_region_jp_pixmap = QIcon(QStringLiteral(":/icons/flag-jp.png")).pixmap(QSize(42, 30));
-	m_region_us_pixmap = QIcon(QStringLiteral(":/icons/flag-us.png")).pixmap(QSize(42, 30));
-	m_region_other_pixmap = QIcon(QStringLiteral(":/icons/flag-other.png")).pixmap(QSize(42, 30));
+
+for (u32 i = 0; i < static_cast<u32>(GameList::Region::Count); i++) 
```

Comment:
Indentation here.

Reply:
Done

Reply:
No it isn't?

Reply:
Oh I removed the indentation

Reply:
Before: 
![image](https://user-images.githubusercontent.com/24227051/171878175-22cea5c8-2a21-4057-9919-097261f68112.png)

After:
![image](https://user-images.githubusercontent.com/24227051/171878236-cdba24b8-6940-43c3-a18a-46537fc39752.png)

Reply:
Then I don't know why Stenzek asked me to change it but ok.

Reply:
Indented properly hopefully